internal/types/host/keepalived_host_collector_host.go

```diff
@@ -97,6 +104,22 @@ func (k *KeepalivedHostCollectorHost) getKeepalivedVersion() (*version.Version,
 	return utils.ParseVersion(stderr.String())
 }
 
+func isEnableJsonSupported() (bool, error) {
```

Comment:
`JSON` should be all caps. https://go.dev/talks/2014/names.slide#5

Reply:
Okay, I'll modify it later.